docs/Configuration_eXo_Platform.rst

```diff
@@ -133,6 +134,32 @@ for the list of applications installed in a space.
       <type>org.exoplatform.social.core.space.impl.SpaceServiceImpl</type>
     </component>
 
+.. _PLFRefGuide.Configurations.Components.Social.SpaceTemplateService:
+
+Space Template Service
+--------------
+
+
+The service is used for space templates management, including creating space templates, extending space templates
+and installing applications. See
+:ref:`Managing Space Templates <SpaceTemplates>`
+
+**Sample configuration**:
+
+.. code:: xml
+
+    <component>
+    <key>org.exoplatform.social.core.space.spi.SpaceTemplateService</key>
+    <type>org.exoplatform.social.core.space.impl.SpaceTemplateServiceImpl</type>
+    <init-params>
+      <value-param>
+        <name>defaultSpaceTemplate</name>
+        <value>community</value>
+      </value-param>
+    </init-params>
+
+*defaultSpaceTemplate* parameter defines the default space template name.
```

Comment:
Could you please add value param in table format like done for other init-params.

Reply:
ok